### PR TITLE
Cleanup unused generated session quotes on PDP

### PR DESCRIPTION
**Pull request by johnmccombs1**
Asana task: https://app.asana.com/0/564264490825835/1120842637218639

### CODE REVIEW

File: app/code/community/Bolt/Boltpay/sql/bolt_boltpay_setup/upgrade-1.4.1-1.4.1.1.php [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+ */
+$installer = Mage::getResourceModel('sales/setup', 'sales_setup');
+$installer->startSetup();
+
+Mage::log('Installing Bolt 1.4.1.1 updates', null, 'bolt_install.log');
+
+if (!$installer->getConnection()->tableColumnExists($installer->getTable('sales/quote'), "is_from_pdp")) {
+    $installer->addAttribute(
+        "quote",
+        "is_from_pdp",
```
Comment (daisy1754) [inline app/code/community/Bolt/Boltpay/sql/bolt_boltpay_setup/upgrade-1.4.1-1.4.1.1.php]: let's add "bolt" so it's clear this is from bolt plugin. how about `is_bolt_pdp`?

File: app/code/community/Bolt/Boltpay/sql/bolt_boltpay_setup/upgrade-1.4.1-1.4.1.1.php [hunk 2]
```diff
@@ -0,0 +1,39 @@
+<?php
```
Comment (daisy1754) [inline app/code/community/Bolt/Boltpay/sql/bolt_boltpay_setup/upgrade-1.4.1-1.4.1.1.php]: can you confirm if this script works when we upgrade 1.4.1 to 2.0.0 as well?

Comment (johnmccombs1) [reply app/code/community/Bolt/Boltpay/sql/bolt_boltpay_setup/upgrade-1.4.1-1.4.1.1.php]: Yes

File: app/code/community/Bolt/Boltpay/sql/bolt_boltpay_setup/upgrade-1.4.1-1.4.1.1.php (hunk 1 above)
Comment (johnmccombs1) [reply app/code/community/Bolt/Boltpay/sql/bolt_boltpay_setup/upgrade-1.4.1-1.4.1.1.php]: Updated.

**Comment by johnmccombs1** [comment]
> @johnmccombs1
> This is a great cron job extension. In your `upgrade-1.4.1-1.4.1.1.php`, I'm thinking that it is a good idea to add a one time cleanup code for all product page orders that exist in the system prior to the new field that you are adding. I think it is a pretty safe bet for quotes that are saved with an empty cart. It will not affect any current sessions.

@mccottry I'm going to create a one time use script to handle this case so we don't end up with code that always wipes out empty quotes for new merchants. I've created a separate task for this. Let me know if you need anything else for PR.

**Pull request merged**